### PR TITLE
New Vel monitor #11

### PR DESCRIPTION
**Pull request by wojtek20032**

### CODE REVIEW

File: Core/MqttClasses.cs [hunk 1]
```diff
@@ -57,6 +57,28 @@ public class RoverControl
...
+			public double CurrentIn { get; set; }
+			public double PidPos { get; set; }
+			public double Tachometer { get; set; }
+			public double VoltsIn { get; set; }
+			public double ADC1 { get; set; }
+			public double ADC2 { get; set; }
+			public double ADC3 { get; set; }
+			public double PPM { get; set; }
+			public long Timestamp { get; set; }
+	}
```
Comment (wisniax) [inline Core/MqttClasses.cs:81]: Inconsistent spacing and indentation

File: MVVM/View/MainView.tscn
Comment (wisniax) [inline MVVM/View/MainView.tscn:1]: Needs to be updated to current master, and then changed...

File: MVVM/View/VelMonitor.tscn
Comment (wisniax) [inline MVVM/View/VelMonitor.tscn:1]: Not sure whether this scales with resolution change

File: MVVM/ViewModel/VelMonitor.cs
Comment (wisniax) [inline MVVM/ViewModel/VelMonitor.cs:1]: Formatting? Spacings? Please follow the previous convention.
Also doesn't seem to support 4 wheels just yet, unless i'm mistaken

**Review by wisniax — CHANGES_REQUESTED** [review]
Missing settings integration for wheel's ID setup + comments

**Comment by AlvaroBajceps** [comment]
Closing due to no activity.

**Pull request closed without merging**